modin/core/execution/dask/common/utils.py

```diff
@@ -59,7 +59,11 @@ def _disable_warnings():
 
         # when the client is initialized, environment variables are inherited
         with set_env(PYTHONWARNINGS="ignore::FutureWarning"):
-            client = Client(n_workers=num_cpus, memory_limit=worker_memory_limit)
+            client = Client(
+                n_workers=num_cpus,
+                threads_per_worker=1,
```

Comment:
Why?

Reply:
On the benchmark I am measuring read_csv is significantly slower with the default client rather than with the configured one. But we can get it in a separate PR as the change is not related to this PR.

```bash
read_csv: 13.72040319442749  # Client()
read_csv: 3.934723377227783  # Client(n_workers=16, threads_per_worker=1)
```